QuizBowlDiscordScoreTracker/Commands/GeneralCommandHandler.cs

```diff
@@ -83,6 +83,8 @@ public Task JoinTeamAsync(string teamName)
                     $@"Couldn't join team ""{teamName}"". Make sure it is not misspelled.");
             }
 
+            string teamId = game.TeamManager.GetTeamIdOrNull(this.Context.User.Id).Result;
```

Comment:
Convert this method to async, then await on these two methods. Better to use async/await than make this synchronous.